app/pathogen/arbovirus/dashboard/filters.tsx

```diff
@@ -288,10 +319,15 @@ export function Filters(props: FiltersProps) {
           filters={filterData.arbovirusFilterOptions}
           data={data}
         />
+        <MapArbovirusFilter 
+          data={data} 
+          state={state} 
+          className={"p-0"} 
+        />
```

Comment:
This is what the ticket requests, but this filter looks awful here. Much better at the top of the filters bar in my opinion. Give me permission and I'll move it.

Reply:
I moved it!